docker-compose.yml

```diff
@@ -101,7 +101,7 @@ mapfishapp:
   image: georchestra/mapfishapp:latest
   volumes:
     - /etc/georchestra:/etc/georchestra
-    - mapfishapp_uploads:/var/local/uploads
+    - mapfishapp_uploads:/mnt/mapfishapp_uploads
```

Comment:
`/var/local/uploads` or `/mnt/mapfishapp_uploads` are not used in code or configuration. Maybe we need to add some configuration ?